engine/src/main/resources/templates/Inventory/dailyEmailBody.html

```diff
@@ -35,11 +35,11 @@
                     </tr>
                 </thead>
                 <tbody>
-                    {#for key in action.context.inventory.errors.keySet()}
-                    {#if key_count < 11}
+                    {#for error in action.context.inventory.errors}
+                    {#if count < 11}
```

Comment:
I'm not sure `count` exists here.

What do you want to count and what is the expected output depending on the result of that count?

Reply:
I want to only show the first 10 hosts, so that the email doesn't get too big.

Reply:
I read it here: https://github.com/quarkusio/qute/blob/master/guide.adoc

Reply:
It looks like you can also use `take` see [docs](https://es.quarkus.io/guides/qute-reference#arrays)

`{#for error in action.context.inventory.errors.take(10)}` and remove the `count`  which also [looks](https://es.quarkus.io/guides/qute-reference#loop_section) valid.

Reply:
Yeah, that seems like a better synthax